README.md

```diff
@@ -67,14 +67,16 @@ const slides = [
 ];
 
 export default class App extends React.Component {
-  this.state = {
```

Comment:
The example breaks because this.state can only be in a constructor, otherwise its a private class variable